kinder/pkg/cluster/status/node.go

```diff
@@ -40,17 +40,16 @@ type commandMutator = func(*cmd.ProxyCmd) *cmd.ProxyCmd
 // Node defines a K8s node running in a kinde(er) docker container or a container hosting
 // one external dependency of the cluster, like etcd or the load balancer.
 type Node struct {
-	name              string
-	role              string
-	ports             map[int32]int32
-	ipv4              string
-	ipv6              string
-	kubernetesVersion string
-	cri               ContainerRuntime
-	kubeadmVersion    *K8sVersion.Version
-	etcdImage         string
-	skip              bool
-	commandMutators   []commandMutator
+	name            string
+	role            string
+	ports           map[int32]int32
+	ipv4            string
+	ipv6            string
+	cri             ContainerRuntime
+	kubeadmVersion  *K8sVersion.Version
```

Comment:
I'm going to open an issue to remove also this from the cache, so everything can change on a node is not cached anymoore